internal/services/compute/windows_virtual_machine_scale_set_extensions_test.go

```diff
@@ -1129,8 +1129,8 @@ resource "azurerm_windows_virtual_machine_scale_set" "test" {
   extension {
     name                       = "AzurePolicyforWindows"
     publisher                  = "Microsoft.GuestConfiguration"
-    type                       = "ConfigurationforLinux"
-    type_handler_version       = "1.0"
+    type                       = "ConfigurationforWindows"
+    type_handler_version       = "1.29"
```

Comment:
This is windows vmss, needs to use `ConfigurationforWindows`, and type 1.0 is not available any more